src/components/modals/ExportModal.jsx

```diff
@@ -11,206 +11,8 @@ import Modal from './Modal'
 import MdFileDownload from 'react-icons/lib/md/file-download'
 import TiClipboard from 'react-icons/lib/ti/clipboard'
 import style from '../../libs/style'
-import GitHub from 'github-api'
-import { CopyToClipboard } from 'react-copy-to-clipboard'
 
 
-class Gist extends React.Component {
```

Comment:
Hi, this code seemed depreciated but the ability to save a style is a nice feature. Is there any plan to add such a feature in the future? For instance, let's say I wanted to save a style to Github for instance.

Reply:
Hi @loicgasser, so this used to exist but then GitHub changed their API removing anonymous gists. Keep an eye on issue #155 for any updates

Reply:
Thanks for the pointer